services/director/src/simcore_service_director/producer.py

```diff
@@ -129,6 +131,13 @@ async def _read_service_settings(
     return settings
 
 
+_SIMCORE_RUNTIME_DOCKER_LABEL_PREFIX: str = "io.simcore.runtime."
+
+
+def _to_simcore_runtime_docker_label_key(key: str) -> str:
+    return f"{_SIMCORE_RUNTIME_DOCKER_LABEL_PREFIX}{key.replace('_', '-').lower()}"
```

Comment:
See above, this is good but knowledge about why we replace might fade if there is no comment

Reply:
ok. this one is special. that is the director-v0 which is scheduled to be retired. I do not want to invest to much time there, I just made it compatible. Otherwise I would agree.